tests/queries/0_stateless/01259_combinator_distinct.sql

```diff
@@ -12,3 +12,12 @@ EXPLAIN SYNTAX SELECT countIf(DISTINCT number % 10, number % 5 = 2) FROM numbers
 
 SELECT sumIf(DISTINCT number % 10, number % 5 = 2) FROM numbers(10000);
 EXPLAIN SYNTAX SELECT sumIf(DISTINCT number % 10, number % 5 = 2) FROM numbers(10000);
+
+SET optimize_rewrite_count_distinct_if = FALSE;
```

Comment:
Better to add new test, without modifying old tests. It is necessary for us to simplify tracking if some test becomes flacky.